presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1404,6 +1413,28 @@ void PrestoServer::reportNodeStatus(proxygen::ResponseHandler* downstream) {
   http::sendOkResponse(downstream, json(fetchNodeStatus()));
 }
 
+void PrestoServer::handleGracefulShutdown(const std::vector<std::unique_ptr<folly::IOBuf>>& body, proxygen::ResponseHandler* downstream){
```

Comment:
The format check is failing. Please fix this (see the Makefile having a format-fix target).

Reply:
Done.